invenio_rdm_records/contrib/journal/custom_fields.py

```diff
@@ -12,13 +12,79 @@
 - journal.volume
 """
 
+from abc import ABC
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services.custom_fields import BaseCF
-from invenio_records_resources.services.records.facets import CFTermsFacet
-from marshmallow import fields
+from marshmallow import fields, missing
 from marshmallow_utils.fields import SanitizedUnicode
 
 
+class PostDumpMixin(ABC):
```

Comment:
To be moved to a more relevant place when the prototype is accepted

Reply:
Suggestions for the placement of `PostDumpMixin`?

Also, to keep the file clean should we move the processors to their own file? E.g. `invenio_rdm_records/contrib/journal/serializer.py` or similar?